src/main/java/org/apache/commons/csv/Lexer.java

```diff
@@ -367,8 +367,7 @@ private Token parseEncapsulatedToken(final Token token) throws IOException {
                         }
                         if (!Character.isWhitespace((char)c)) {
                             // error invalid char between token and next delimiter
-                            throw new IOException("(line " + getCurrentLineNumber() +
-                                    ") invalid char between encapsulated token and delimiter");
+                            throw new IOException("Invalid char between encapsulated token and delimiter at line: " + getCurrentLineNumber() + ", position: " + getCharacterPosition());
```

Comment:
This probably shouldn't be an IOException but that issue is not new with this PR